app/build.gradle.kts

```diff
@@ -45,4 +49,18 @@ dependencies {
     testImplementation(libs.junit)
     androidTestImplementation(libs.androidx.junit)
     androidTestImplementation(libs.androidx.espresso.core)
+
+    //Retrofit
+    implementation(libs.retrofit)
+    implementation(libs.converter.gson.v2110)
```

Comment:
I advise you to use Moshi instead of GSON, moshi auto generated the adapters, but gson use reflection, that is, Gson will crate the classes when the app is running -> deacresed perfomance.